pkgs/by-name/re/ren-find/package.nix

```
@@ -0,0 +1,26 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ren-find";
+  version = "unstable-2024-01-11";
```

```suggestion
  version = "0-unstable-2024-01-11";
```

version should start with a digit.